grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlers.java

```diff
@@ -110,7 +114,8 @@ static UnframedGrpcErrorHandler of(UnframedGrpcStatusMappingFunction statusMappi
      * @param statusMappingFunction The function which maps the {@link Throwable} or gRPC {@link Status} code
      *                              to an {@link HttpStatus} code.
      */
-    static UnframedGrpcErrorHandler ofJson(UnframedGrpcStatusMappingFunction statusMappingFunction) {
+    static UnframedGrpcErrorHandler ofJson(
+            UnframedGrpcStatusMappingFunction statusMappingFunction, MessageMarshaller jsonMarshaller) {
```

Comment:
Optional) This class becomes larger and more complex compared to its initial implementation. I think it's time to refactor. Are you interested in sending a follow/up PR to extract the code into `TextUnframedGrpcErrorHandler` and `JsonUnframedGrpcErrorHandler`?

Reply:
Of course!